### ISSUE TITLE
Make it possible to edit the size field

### ISSUE THREAD

**Issue by PVince81** [owncloud/documents#290]
Currently if you want to quickly make your text big, you need to click the up arrow in the "font size" field of the toolbar.

I'd like to be able to change the value from that field manually by clicking inside and typing a value.

@owncloud/designers

**Comment by VicDeo** [comment]
@jancborchardt Thanks for triaging but I hope you don't expect me to rewrite half of WebODF for 9.1.
Most of these `bugs` are features that are not implemented upstream.

**Comment by jancborchardt** [comment]
@VicDeo no, no worries. I don’t know what exactly is already implemented.
